WordPress/Classes/Utility/HelpshiftUtils.m

```diff
@@ -107,6 +109,20 @@ + (void)refreshUnreadNotificationCount
     [HelpshiftSupport getNotificationCountFromRemote:YES];
 }
 
++ (NSArray<NSString *> *)planTagsForAccount:(WPAccount *)account
+{
+    NSMutableSet<NSString *> *tags = [NSMutableSet set];
+    for (Blog *blog in account.blogs) {
+        if (blog.planID == nil) {
+            continue;
+        }
+        NSString *tag = [NSString stringWithFormat:@"plan:%@", blog.planID];
+        [tags addObject:tag];
+    }
+
+    return [tags allObjects];
```

Comment:
Since we're using IDs, that would mean hardcoding which IDs mean free. Also there's really no value in removing them. Initially I planned to create a Free smart view searching for "Free", but you can also do that by searching issues that don't have a paid tag.